orienteer-core/src/main/java/org/orienteer/core/widget/AbstractWidget.java

```diff
@@ -232,4 +235,24 @@ protected ODatabaseDocument getDatabase() {
 	protected OSchema getSchema() {
 		return OrientDbWebSession.get().getSchema();
 	}
+
+	protected void removeCommandById(String id) {
+		commands.remove(id);
+	}
+
+	protected String getSettingsCommandId() {
+		return settingsCommandId;
+	}
+
+	protected String getHideCommandId() {
+		return hideCommandId;
+	}
+
+	protected String getDeleteCommandId() {
+		return deleteCommandId;
+	}
+
+	protected String getFullScreenCommandId() {
```

Comment:
I don't like this kind of solution... It's expected that commands are various, but here particular methods are pre-wared to set of commands... What's about find child by class and replace it?